### PR TITLE
Add install instructions for HSL on windows 

### PR DESCRIPTION
**Pull request by avdudchenko**
This adds simple instructions on how to add HSL binaries to windows conda install of cyipopt.  - tested only on my machine, so hopefully it works for others.

### CODE REVIEW

File: docs/source/install.rst [hunk 1]
```diff
@@ -414,3 +417,39 @@ name must be specified because the default name ipopt looks for is
...
+This will download a zipped file that contains a folder ``bin``. Copy all the DLL files from
+that folder into your conda env\Library\bin folder 
+(This folder should also contain the an ipopt dll file installed with cyipopt (in our case it was called ``ipopt-3.dll``)::
+
+   <conda_location>\envs\<env_name>\Library\bin
+
+Once the DLL files are placed you should be able to access the HSL solvers using solver options, in pyomo::
+
+   solver = pyo.SolverFactory("cyipopt")
+   solver.config.options["linear_solver"] = "ma27"
```
Comment (moorepants) [inline docs/source/install.rst:442]: pyomo is a different library, so can you change this to: ` problem.add_option('linear_solver', 'ma57')` which is the syntax of cyipopt.

**Comment by moorepants** [comment]
One comment for adjustment, but otherwise thanks for the helpful addition!

File: docs/source/install.rst (hunk 1 above)
Comment (avdudchenko) [reply docs/source/install.rst]: Done.

**Comment by moorepants** [comment]
The test failures are because scipy 1.13 broke things. If you change to `scipy>1.8.0,<1.13` to `.github/workflows/tests.yml`:

https://github.com/mechmotum/cyipopt/blob/master/.github/workflows/tests.yml#L58

the tests should pass.

**Comment by avdudchenko** [comment]
> The test failures are because scipy 1.13 broke things. If you change to `scipy>1.8.0,<1.13` to `.github/workflows/tests.yml`:
> 
> https://github.com/mechmotum/cyipopt/blob/master/.github/workflows/tests.yml#L58
> 
> the tests should pass.

I tried that, but not sure where I am configuring it wrong (says file not found). ( I never really worked with github workflows or mamba.) also trying scipy>1.8.0,<1.13.0. but doubt that is the issue.

**Comment by moorepants** [comment]
You don't have to debug that. I'll work on it.

**Pull request merged**